usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -35,7 +36,12 @@
 # python manage.py es_rapidloader --index-name NEWINDEX --reload-all all
 
 DEFAULT_DATETIME = datetime.strptime("2007-10-01+0000", "%Y-%m-%d%z")
-ES = Elasticsearch(settings.ES_HOSTNAME, timeout=300)
+
+if 'https' in settings.ES_HOSTNAME:
+    ES = Elasticsearch(settings.ES_HOSTNAME, timeout=300, use_ssl=True, verify_certs=True, ca_certs=certifi.where())
+else:
+    ES = Elasticsearch(settings.ES_HOSTNAME, timeout=300)
```

Comment:
Is the timeout for creating the ES or for each query? At 5 minutes no user will still be waiting for a response.

Reply:
This is an ETL script, not API